File: RentACarApplication/src/app/car/dtos/all-cars.dto.ts
import { RegionEnum } from "../../enums/region-enum";
import { CarImage } from "./image.dto";

export class AllCarsDto{
  id: number
  make: string;
  model: string;  
  price: number;
  region: RegionEnum; 
  isRented: boolean;
  imageFiles: File;
  images: CarImage[];
}

File: RentACarApp/Dtos/CarDtos/AllCarsDto.cs
using RentACarApp.Database.Enums;
using RentACarApp.Database.Models;

namespace RentACarApp.Dtos.CarDtos
{
    public class AllCarsDto
    {
        public int Id { get; set; }
        public string Make { get; set; }

        public string Model { get; set; }

        public decimal Price { get; set; }

        public RegionEnum Region { get; set; }

        public bool IsRented { get; set; }

        public List<IFormFile> ImageFiles { get; set; }

        public List<Image> Images { get; set; } = new List<Image>();

    }
}
